.gitignore

```diff
@@ -1,3 +1,4 @@
+/composer.lock
```

Comment:
The `composer.lock` file is required because it locks down the versions concise depends on for its CI build system. If it were removed you would not be able to go back to a previous commit and test the code under the same circumstances.

https://getcomposer.org/doc/02-libraries.md#lock-file